sympy/calculus/util.py

```diff
@@ -1128,14 +1128,15 @@ def __add__(self, other):
             if other is S.Infinity and self.min is S.NegativeInfinity or \
                     other is S.NegativeInfinity and self.max is S.Infinity:
                 return AccumBounds(-oo, oo)
-            elif other.is_extended_real and self.min is not S.NegativeInfinity and self.max is not S.Infinity:
-                return AccumBounds(Add(self.min, other), Add(self.max, other))
-            elif self.min is S.NegativeInfinity and self.max is S.Infinity:
-                return AccumBounds(-oo, oo)
-            elif self.min is S.NegativeInfinity:
-                return AccumBounds(-oo,self.max+other)
-            elif self.max is S.Infinity:
-                return AccumBounds(self.min+other,oo)
+            elif other.is_extended_real:
+                if self.min is not S.NegativeInfinity and self.max is not S.Infinity:
+                    return AccumBounds(Add(self.min, other), Add(self.max, other))
```

Comment:
Could this be the last `else` line?

Reply:
Does this block need a else statement ? 
As all of the cases are covered in elif statements.

Reply:
This line can be run when none of the three other conditions are true.